docs/src/components/Pre.tsx

```diff
@@ -0,0 +1,97 @@
+import {Button, CopyToClipboard} from "nextra/components";
+import {WordWrapIcon} from "nextra/icons";
+import type {ComponentProps, ReactElement} from "react";
+import {useCallback, useRef} from "react";
+import {Javascript, Json, Typescript} from "./icons";
+
+function cn(...classes: Array<string | boolean | undefined>) {
+  return classes.filter(Boolean).join(" ");
+}
+
+export const Pre = ({
+  children,
+  className,
+  hasCopyCode,
+  filename,
+  ...props
+}: ComponentProps<"pre"> & {
+  filename?: string;
+  hasCopyCode?: boolean;
+}): ReactElement => {
```

Comment:
```suggestion
}: ComponentProps<"pre"> & {
  filename?: string;
  hasCopyCode?: boolean;
  "data-language"?: string 
}): ReactElement => {
```

to skip the `as`' below?